src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Tag,
+  Button,
+  TextInput,
+  Select,
+  Checkbox,
+  IconSearch,
+} from "hds-react";
+
+import Text from "../../text/Text";
+import {
+  mockSportTypes,
+  mockArea,
+  mockPrice,
+  mockCategory,
+} from "./tmp/mockedData";
+import styles from "./searchPageSearchForm.module.scss";
+
+function SearchPageSearchForm() {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className={styles.searchArea}>
+      <Text variant="h1">Mitä etsit?</Text>
+      <form role="search" className={styles.form} onSubmit={handleSubmit}>
+        <TextInput
+          placeholder="Kirjoita hakusana, esim. uimahalli tai jooga"
+          id="text"
+          name="text"
+          className={styles.inputWithIcon}
+        >
+          <IconSearch aria-hidden="true" />
+        </TextInput>
+        <Select
+          label=""
```

Comment:
Missing label. There should be a prop for visually hiding the label.

Reply:
Hmm, seems that the `hideLabel` prop is only for TextInput 🤔